### PR TITLE
fix: update URLs for Stakater and centralized cost monitoring docs page

### PR DESCRIPTION
**Pull request by seialkali**
Closes D2IQ-91912

**What problem does this PR solve?**:
Updates the links to the centralized cost monitoring docs page to the latest version, as well as Stakater Open Source

**Which issue(s) does this PR fix?**:
https://jira.d2iq.com/browse/D2IQ-91912

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**If the PR adds a version bump, does it add a breaking change in License**:

- [ ] No License Change (or NA).

### CODE REVIEW

**Review by TattdCodeMonkey — APPROVED** [review]

**Review by tillt — APPROVED** [review]

**Review by tillt — APPROVED** [review]

**Review by gracedo — APPROVED** [review]

File: services/kubecost/metadata.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ overview: |-
 
   ## More Information
   - [kubecost.com](https://www.kubecost.com/)
-  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/kommander/latest/monitoring/centralized-cost-monitoring/)
+  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/2.3/centralized-cost-monitoring)
```
Comment (takirala) [inline services/kubecost/metadata.yaml:33]:
```suggestion
  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/latest/centralized-cost-monitoring)
```
or `2.4` perhaps - cuz `main` is ahead of `2.3` ?

**Review by takirala — APPROVED** [review]

File: services/kubecost/metadata.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ overview: |-
 
   ## More Information
   - [kubecost.com](https://www.kubecost.com/)
-  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/kommander/latest/monitoring/centralized-cost-monitoring/)
+  - [Centralized Cost Monitoring](https://docs.d2iq.com/dkp/latest/centralized-cost-monitoring)
```
Comment (vacas5) [inline services/kubecost/metadata.yaml:33]: what if a user on 2.4 gets sent to our docs for 2.6 sometime in the future? Is that what we want?

Comment (tillt) [reply services/kubecost/metadata.yaml:33]: Very good question and our intent of the old URL was just that. Not sure if things changed on that end.

Comment (tillt) [reply services/kubecost/metadata.yaml]: New approach is to pin, @msdolbey  made clear to me how important that change is.

We already landed the 2.3 release branch version, pointing to `2.3`; https://github.com/mesosphere/kommander-applications/pull/504

By now using `2.4` here, we are carrying forward the intent to pin this URL. that in turn means we will have to update that one with every major release.

**Comment by msdolbey** [comment]
Thank you so much @seialkali !

**Pull request merged**